addons/account_bank_statement_import/account_bank_statement_import.py

```diff
@@ -96,7 +88,8 @@ def _parse_file(self, cr, uid, data_file, context=None):
         """
         raise UserError(_('Could not make sense of the given file.\nDid you install the module to support this type of file ?'))
 
-    def _check_parsed_data(self, cr, uid, stmts_vals, context=None):
+    @api.multi
```

Comment:
No need for a decorator since those methods are never called in old-api style (they are only used in this class). Will also apply to _parse_file once all bank statement import modules are migrated